proposals/csharp-9.0/function-pointers.md

```diff
@@ -358,9 +358,12 @@ the attribute:
 
 * It is an error to directly call a method annotated with this attribute from C#. Users must obtain a function pointer to
 the method and then invoke that pointer.
-* It is an error to apply the attribute to anything other than a static method. The C# compiler will mark any non-static
-methods imported from metadata with this attribute as unsupported by the language.
-* It is an error to have managed types as parameters or the return type of a method marked with the attribute.
+* It is an error to apply the attribute to anything other than an ordinary static method or ordinary static local function.
+The C# compiler will mark any non-static methods imported from metadata with this attribute as unsupported by the language.
+* It is an error to have managed types as parameters or the return type of a method marked with the attribute. This includes
+types with type parameters, even if those type parameters are constrained to `unmanaged` or substituted with `unmanaged` types.
```

Comment:
The type parameter adjustment is to prevent things like `int?` being parameters to one of these methods.